### DISCUSSION TITLE
Welcome to 🐻 Bearmentor Discussions!

### DISCUSSION THREAD

**Discussion by bear-mentor** [bearmentor/bearmentor-com-legacy#1] [Announcements]
> Brilliant mentoring

🐻 [Bearmentor](https://bearmentor.com) is a place for tech and creative professionals to mentor each others.

While this GitHub Discussions is a place to get help, offer solutions, share your ideas, and give feedback for the 🐻 [Bearmentor](https://bearmentor.com) platform.

We're building the open mentoring platform for people and organization. Open source, free to use, with premium options. A true mentorship should enable to teach, guide, and share in long term growth.

We allow you to:

- Connect to prospective mentors and mentees
- Explore various learning programs
- Filter out for serious commitment
- Design custom mentorship experience
- Set achievable goals
- Get paid to mentor people
- Adjust your own rate

Check out 🐻 [Bearmentor.com](https://bearmentor.com)

This is a community we build together. Please be welcoming and open minded.

Thank you and enjoy!

---

Available categories on this 🗨️ Discussions:

- 📣 [Announcements](https://github.com/orgs/bearmentor/discussions/categories/announcements) - Find announcements, changes, and requests for feedback.
- 🙏 [Help](https://github.com/orgs/bearmentor/discussions/categories/help) - Get help with your project or a question about the platform.
- 🐛 [Bug Report](https://github.com/orgs/bearmentor/discussions/categories/bug-report) - Tell us about things that aren’t quite right like typos, broken feature, or unexpected behavior.
- 💡 [Ideas](https://github.com/orgs/bearmentor/discussions/categories/ideas) - Share ideas for features and changes you would like to see.
- 💬 [Show, Tell & Talk](https://github.com/orgs/bearmentor/discussions/categories/show-and-tell) -  If you have a story that you'd like to share or show off, please do!

**Comment by mhaidarhanif** [comment]
Hello @bear-mentor and welcome everybody! 🙌

**Reply by amadzuki** [reply]
🔥 🔥 🔥
